shell/platform/fuchsia/flutter/tests/fakes/scenic/fake_flatland.h

```diff
@@ -82,6 +82,12 @@ class FakeFlatland
 
   const FakeGraph& graph() { return current_graph_; }
 
+  const std::unordered_map<uint64_t,
+                           std::vector<fuchsia::ui::composition::HitRegion>>&
+  hit_regions() {
```

Comment:
See other comment.  You shouldnt need this if adding to FakeTransform

Reply:
Done